docs/user/exams/instructors_guide.rst

```diff
@@ -0,0 +1,192 @@
+===================
+Instructors’ Guide
+===================
+
+.. _timeline:
+
+Timeline of an Artemis Online Exam
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    .. _exam_creation_and_configuration:
+
+- **Exam Creation and Configuration**
```

Comment:
please use numbering 1., 2., ... here

Reply:
and use the same numbering scheme in the headlines below for easier navigation

Reply:
this means, please put all sections below as sub sections into one of these new main 6 sections.